install/kubernetes/helm/istio/values.yaml

```diff
@@ -395,10 +395,12 @@ tracing:
   enabled: false
   jaeger:
     enabled: false
+    memory:
+      max_traces: 50000
   replicaCount: 1
   image:
     repository: jaegertracing/all-in-one
-    tag: 1.4.1
+    tag: 1.5.0
```

Comment:
How about use `1.5` instead of `1.5.0`? Thus we will always get the latest stable version for `1.5.x`

Reply:
Good point - although I guess 1 would be even better.

Reply:
Ok changed to 1.5, as better to be consistent.

Reply:
I think we want to use `x.x` is because if you use `x`, there maybe some unstable versions such as `beta` or `alpha` etc.